pkg/kudoctl/kudoinit/manager/manager.go

```diff
@@ -118,8 +118,11 @@ func (m *Initializer) verifyManagerInstalled(client *kube.Client, result *verifi
 		result.AddWarnings(fmt.Sprintf("KUDO manager has an unexpected image. Expected %s but found %s", m.options.Image, mgr.Spec.Template.Spec.Containers[0].Image))
 		return nil
 	}
-	if err := health.IsHealthy(mgr); err != nil {
-		result.AddErrors("KUDO manager seems to be not healthy")
+	if healthy, msg, err := status.IsHealthy(mgr); !healthy || err != nil {
+		if err != nil {
+			return err
+		}
+		result.AddErrors("KUDO manager seems to be not healthy: %s", msg)
```

Comment:
Not yours and nitpicky, but let's change this message because the KUDO manager _is_ unhealthy here.
```suggestion
		result.AddErrors("KUDO manager is not healthy: %s", msg)
```